test/issue-65.test.ts

```diff
@@ -0,0 +1,103 @@
+import { PolymorphicSerializer, Relator, Serializer } from '../lib';
+import ResourceIdentifier from '../lib/models/resource-identifier.model';
+import Resource from '../lib/models/resource.model';
+
+describe('Issue #65 - Polymorphic relator', () => {
+  class Model {
+    id: string;
+
+    children: Child[];
```

Comment:
By the way, congratulations on your first child! I saw you replied on one of the issues and I wanted to congratulate you :)

Reply:
Thank you! Still getting used to the change in sleep patterns, but loving it :D